src/ert/parsing/config_errors.py

```diff
@@ -17,3 +19,14 @@ def __init__(self, errors: str, config_file: Optional[str] = None) -> None:
             if self.config_file
             else f"{self.errors}"
         )
+
+    @classmethod
+    def raise_from_collected(cls, collected_errors: List[ErrorInfo]):
+        if len(collected_errors) > 0:
```

Comment:
For the GUI we need the list of errors as separate boxes, so I suggest going back to the approach of keeping the list of errors as a field.